apps/demo/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src"]
+  "include": ["src", "*.js", ".*.cjs"]
```

Comment:
Dot files have to be specified explicitly. Advanced glob patterns like `{,.}*.?(c)js` are not supported, unfortunately.